packages/wrangler/src/pages/dev.tsx

```diff
@@ -402,6 +394,8 @@ async function spawnProxyProcess({
 
 	proxy.on("close", (code) => {
 		logger.error(`Proxy exited with status ${code}.`);
+		CLEANUP();
+		process.exit(code ?? 0);
```

Comment:
```suggestion
		process.exitCode = code ?? 0;
```

Reply:
I added this logic in just to make it pass the exit code through in case wrangler is being run by something like nodemon or docker, which may otherwise restart it if it exited with a 0, happy to always make it 0 or whatever though